### PR TITLE
[PARTI-338] Set `force` as false by default

### PR DESCRIPTION
**Pull request by pedro-lb**
# PR Description

This PR adds the following changes:

- Sets the `force` param to default to false

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@constructor-io/catalog-ingestor",
-  "version": "1.3.1",
+  "version": "1.3.2",
```
Comment (guiquintelas) [inline package.json:3]: **Suggestion**: I feel see this as a breaking change, very minor but still breaking :/ WDYT?

**Review by guiquintelas — APPROVED** [review]
Very good work here pedro! 

Left a couple of comments there, but non-blocking

File: package.json (hunk 1 above)
Comment (pedro-lb) [reply package.json:3]: Hmmmm after thinking a bit more, I agree. Let's do 1.4.x

File: src/catalogIngestor/index.ts
[the remark this answers is not in the document]
Comment (pedro-lb) [reply src/catalogIngestor/index.ts:1]: Yep! we should. Here's the new version:

image

4066bfd8b6d0b2cfd4d4335389b2e8cde96bcbf6

File: package.json (hunk 1 above)
Comment (pedro-lb) [reply package.json]: 1dc9d314d4fc675c80ef6c662f66baa399eecb4a

**Pull request merged**